src/sentry/api/endpoints/project_rule_actions.py

```diff
@@ -58,7 +59,7 @@ def post(self, request: Request, project) -> Response:
                 "frequency": 30,
             }
         )
-        rule = Rule(id=-1, project=project, data=data)
+        rule = Rule(id=-1, project=project, data=data, label=data.get("name"))
```

Comment:
Rules are invalid without a label, so I feel comfortable including this across the board.